advanced_functionality/mxnet_mnist_byom/mxnet_mnist.ipynb

```diff
@@ -223,9 +225,7 @@
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {
-    "collapsed": true
-   },
+   "metadata": {},
    "outputs": [],
    "source": [
     "predictor = sagemaker_model.deploy(initial_instance_count=1,\n",
```

Comment:
Can you add a cell above this that returns logging back to normal? e.g.

```
import logging
logging.getLogger().setLevel(logging.WARNING)
```

Otherwise, for every "-" in the deploy we get another 20 lines of debug info.